state/postgres_relation_stats.go

```diff
@@ -43,8 +43,20 @@ type PostgresIndexStats struct {
 	IdxBlksHit  int64 // Number of buffer hits in this index
 }
 
+type PostgresColumnStats struct {
+  SchemaName  string
+  TableName   string
+  ColumnName  string
+  Inherited   bool
+  NullFrac    float64
+  AvgWidth    int32
+  NDistinct   float64
+  Correlation null.Float
+}
+
 type PostgresRelationStatsMap map[Oid]PostgresRelationStats
 type PostgresIndexStatsMap map[Oid]PostgresIndexStats
+type PostgresColumnStatsMap map[string][]PostgresColumnStats
```

Comment:
Can we use a more strongly typed key here? (see e.g. `PostgresStatementKey` in `state/postgres_statement.go`)